content/release-notes/1.52.1.md

```diff
@@ -0,0 +1,13 @@
+---
+date: 2021-10-04T08:00:00-07:00
+linktitle: "1.52.1"
+title: "1.52.1"
+kubernetes: "1.19, 1.20, and 1.21"
+---
+
+{{<fixes>}}
+ * Fixed an issue with the [Helm rendering workflow beta](/release-notes/1.47.0/) where private image Kustomize targets inside of Helm charts failed when sub-charts are present.
```

Comment:
```suggestion
 * Fixed an issue with the [Helm rendering workflow beta](/release-notes/1.47.0/) where private image Kustomize targets inside of Helm charts failed when sub-charts were present.
```
I think that's better since "failed" is past tense.